README.rst

```diff
@@ -1,104 +1,138 @@
-| Maple            | >= 6.0                |
-+------------------+-----------------------+
-| Nutmeg           | >= 6.2                |
-+------------------+-----------------------+
-| Olive            | >= 8.0                |
-+------------------+-----------------------+
-| Palm             | >= v10.0.0            |
-+------------------+-----------------------+
-| Quince           | >= v11.0.0            |
-+------------------+-----------------------+
-
-**NOTE**: Since 6.2 version, eox-tenant does not support Django 2.2
-
-The following changes to the plugin settings are necessary. If the release you are looking for is
-not listed, then the accumulation of changes from previous releases is enough.
-
-**Ironwood**
+- **Microsites:** Store the microsite configuration.
+- **Routes:** Configure the URL for a tenant.
+- **Tenant configs:** Store the configuration for each tenant.
+- **Tenant organizations:** Link each organization with one or multiple tenants.
 
-.. code-block:: yaml
+Add ``EDNX_USE_SIGNAL = True`` in each microsite/tenant that wants to use the plugin. 
+ 
+Compatibility Notes
+--------------------
 
-  GET_BRANDING_API: 'eox_tenant.edxapp_wrapper.backends.branding_api_h_v1'
-  GET_CERTIFICATES_MODULE: 'eox_tenant.edxapp_wrapper.backends.certificates_module_i_v1'
-  GET_SITE_CONFIGURATION_MODULE: 'eox_tenant.edxapp_wrapper.backends.site_configuration_module_i_v1'
-  GET_THEMING_HELPERS: 'eox_tenant.edxapp_wrapper.backends.theming_helpers_h_v1'
-  EOX_TENANT_EDX_AUTH_BACKEND: "eox_tenant.edxapp_wrapper.backends.edx_auth_i_v1"
-  EOX_TENANT_USERS_BACKEND: 'eox_tenant.edxapp_wrapper.backends.users_i_v1'
-  EDXMAKO_MODULE_BACKEND: 'eox_tenant.edxapp_wrapper.backends.edxmako_h_v1'
-  UTILS_MODULE_BACKEND: 'eox_tenant.edxapp_wrapper.backends.util_h_v1'
++------------------+----------------+
+| Open edX Release | Version        |
++==================+================+
+| Ironwood         | < 3.0          |
++------------------+----------------+
+| Juniper          | >= 3.0 < 4.0   |
++------------------+----------------+
+| Koa              | >= 4.0 <= 5.1.3|
++------------------+----------------+
+| Lilac            | >= 4.0 < 6.2   |
++------------------+----------------+
+| Maple            | >= 6.0 < 11.6  |
++------------------+----------------+
+| Nutmeg           | >= 6.2         |
++------------------+----------------+
+| Olive            | >= 8.0         |
++------------------+----------------+
+| Palm             | >= v10.0.0     |
++------------------+----------------+
+| Quince           | >= v11.0.0     |
++------------------+----------------+
+
+**NOTE**: Since the 6.2 version, eox-tenant does not support Django 2.2
+
+The plugin is configured for the latest release (Quince). The following changes in the plugin settings should be applied to be used for previous releases.
+
+If the release you are looking for is not listed, the actual configuration is sufficient or it is incompatible with the current eox-tenant version.
```

Comment:
I thought that if wasn't not listed, it's not supported. In what case does the other scenario happen?

Reply:
Please take in mind, that I removed all previous version modifications and listed only the latest (that is why the paragraph mentions the _**the current eox-tenant version**_) to make emphasis (from now and the subsequent releases) the real support for each release, and limit the configuration changes, I mean:

The actual description says 

> If the release you are looking for is not listed, then the accumulation of changes from previous releases is enough.

If I check the changes from Koa to Maple (in the [master branch readme](https://github.com/eduNEXT/eox-tenant/blob/master/README.rst?plain=1#L41)) those are the same, so I'm not accumulating changes, I'm explicitly telling which backends use.

I think using that expression could be confusing, how many previous releases are useful for the accumulation of changes?

Reply:
Mmm, I get it now that you explained it. Back when I left the comment, I was thinking about the ambiguity of "or the configuration is enough or is not supported." How do I know?

Still, this is a non-blocking comment.

Reply:
@dcoa why are we removing the listed settings for the previous releases and only keeping the latest? 
Could you please explain this decision and the reasoning behind it in the cover letter?

Reply:
@magajh I will explain that here and modify the other repositories once we agree on the best way to write this section.